Resources/doc/installation.rst

```diff
@@ -78,6 +78,20 @@ the MongoDB ODM across your application:
     Of course, you'll also need to make sure that the MongoDB server is running
     in the background. For more details, see the MongoDB `Quick Start`_ guide.
 
+
+Authorization
+-------------
+
+If you use authorization on your MongoDB database you can provide username, password, and authentication database in the following way:
+
+    # app/config/parameters.yml
+    parameters:
+        mongodb_server: "mongodb://username:password@localhost:27017/auth-db"
+
+.. note::
+   
+    The authentication database is different to the default database and needs to be provided as MongoDB checks during aurhorization the users collection of that particular database.
```

Comment:
I think this note can be removed if the above changes are made. If kept, I'd suggest rephrasing as:

> The authentication database is different from the default database used by MongoDB ODM.

From an ODM user's respective, I think the concept of a default database is when asking the question "where does my application and model data live?". Also, the relevant collection is [system.users](https://docs.mongodb.com/manual/reference/system-users-collection/index.html) in current server versions, but again that only applies to auth mechanisms that actually use a database in lieu of some external service.